src/cloud/project/new-relic.md

```diff
@@ -175,7 +208,42 @@ All collected data details the time spent on an action transmitting data, querie
 {:.bs-callout-info}
 You can learn more about using the New Relic APM and Infrastructure agents to collect and analyze application data from the [New Relic APM] and [New Relic Infrastructure][New Relic infrastructure] documentation.
 
-## Monitor performance with alert policies
+### View and analyze log data
+
+After you log in to your New Relic account, you can use the New Relic Logs UI to search across the aggregated log data to troubleshoot application, infrastructure, CDN, and WAF errors. Also, you can connect the log data with other data collected by New Relic APM and Infrastructure services to create charts, dashboards, and alerts to manage Magento application and cloud service operations.
+
+{:.procedure}
+To use the New Relic Logs UI:
+
+1. Use your New Relic credentials to [log in to your New Relic account][New Relic login].
+
+1. From the New Relic ONE home page, select **Logs**.
```

Comment:
You may want to reverse these points, I do not believe our customers land on the New Relic ONE home page.  I believe the customer work flow is:
1. Auth
1. Select application
 > Lands them on APM page